From: Lukas Bulwahn <lbulwahn@redhat.com>
Subject: MAINTAINERS: improve entries in CODE TAGGING and MEMORY ALLOCATION PROFILING
Date: Thu, 11 Apr 2024 08:47:17 +0200

Commit 5acf2502db99 ("MAINTAINERS: add entries for code tagging and memory
allocation profiling") adds the two new sections CODE TAGGING and MEMORY
ALLOCATION PROFILING.  The files in these sections however do not match
with the files added in the corresponding patch series.

Improve the two entries to refer to all files added with that series and
drop the entries to non-existing files.

Link: https://lkml.kernel.org/r/20240411064717.51140-1-lukas.bulwahn@redhat.com
Signed-off-by: Lukas Bulwahn <lukas.bulwahn@redhat.com>
Acked-by: Kent Overstreet <kent.overstreet@linux.dev>
Cc: Suren Baghdasaryan <surenb@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/MAINTAINERS~maintainers-add-entries-for-code-tagging-and-memory-allocation-profiling-fix
+++ a/MAINTAINERS
@@ -5256,6 +5256,7 @@ CODE TAGGING
 M:	Suren Baghdasaryan <surenb@google.com>
 M:	Kent Overstreet <kent.overstreet@linux.dev>
 S:	Maintained
+F:	include/asm-generic/codetag.lds.h
 F:	include/linux/codetag.h
 F:	lib/codetag.c
 
@@ -14164,10 +14165,10 @@ M:	Suren Baghdasaryan <surenb@google.com
 M:	Kent Overstreet <kent.overstreet@linux.dev>
 L:	linux-mm@kvack.org
 S:	Maintained
+F:	Documentation/mm/allocation-profiling.rst
 F:	include/linux/alloc_tag.h
-F:	include/linux/codetag_ctx.h
+F:	include/linux/pgalloc_tag.h
 F:	lib/alloc_tag.c
-F:	lib/pgalloc_tag.c
 
 MEMORY CONTROLLER DRIVERS
 M:	Krzysztof Kozlowski <krzk@kernel.org>
_
